src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+.wrapper {
+  width: 100px;
+  height: 100px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Please remove margin-right for each last star
![image](https://github.com/mate-academy/layout_stars/assets/54936044/9d8249a9-48ac-4f75-bf8e-dd30d3808e67)

Reply:
It is still not done